sky/sdk/lib/widgets/variable_height_scrollable.dart

```diff
@@ -14,11 +12,16 @@ class VariableHeightScrollable extends Scrollable {
   VariableHeightScrollable({
     String key,
     this.builder,
-    this.token
-  }) : super(key: key);
+    this.token,
+    this.layoutState
+  }) : super(key: key) {
+    assert(layoutState != null);
```

Comment:
Does this mean clients of VariableHeightScrollable need to pass in a layoutState?  Maybe VariableHeightScrollable could accept a null layoutState and initialize it in initState so that clients that don't need this this level of control can have a simpler API.